front/src/routes/integration/all/zigbee2mqtt/setup-page/SetupTab.jsx

```diff
@@ -0,0 +1,262 @@
+                )}
+                {props.dockerBased && props.networkModeValid && (
+                  <Text id="integration.zigbee2mqtt.setup.enableZigbee2mqtt" />
+                )}
+              </span>
+            </label>
+          </div>
+          <div class="card-header d-none d-sm-block">
+            <h2 class="card-title">
+              <Text id="integration.zigbee2mqtt.setup.serviceStatus" />
+            </h2>
+          </div>
+          <div class="row justify-content-center">
+            <div class="col-auto">
+              <table class="table table-responsive table-borderless table-sm d-none d-sm-block">
+                <thead class="text-center">
+                  <tr>
+                    <th class="text-center">{`Gladys`}</th>
+                    <th class="text-center" />
+                    <th class="text-center">{props.mqttExist && 'MQTT'}</th>
+                    <th class="text-center" />
+                    <th class="text-center">{props.zigbee2mqttExist && 'Zigbee2mqtt'}</th>
+                  </tr>
+                </thead>
+                <tbody class="text-center">
+                  <tr>
+                    <td class="text-center">
+                      <img src={logoGladys} alt={`Gladys`} title={`Gladys`} width="80" height="80" />
+                    </td>
+                    <td class="text-center" style="vertical-align:middle;display:flex;align-items:center;">
+                      {props.mqttRunning && (
+                        <hr
+                          style={{ color: '#00f', backgroundColor: '#00f', borderColor: '#00f', height: 2, width: 40 }}
+                        />
+                      )}
+                      {props.mqttRunning && props.gladysConnected && (
+                        <i style={{ color: '#0f0', fontSize: '24px' }} class="fe fe-check" />
+                      )}
+                      {props.mqttRunning && !props.gladysConnected && (
+                        <i style={{ color: '#f00', fontSize: '24px' }} class="fe fe-x" />
+                      )}
+                      {props.mqttRunning && (
+                        <hr
+                          style={{ color: '#00f', backgroundColor: '#00f', borderColor: '#00f', height: 2, width: 40 }}
+                        />
+                      )}
+                    </td>
+                    <td class="text-center">
+                      {props.mqttExist && <img src={logoMqtt} alt={`MQTT`} title={`MQTT`} width="80" height="80" />}
+                    </td>
+                    <td class="text-center" style="vertical-align:middle;display:flex;align-items:center;">
+                      {props.zigbee2mqttRunning && (
+                        <hr
+                          style={{ color: '#00f', backgroundColor: '#00f', borderColor: '#00f', height: 2, width: 40 }}
+                        />
+                      )}
+                      {props.zigbee2mqttRunning && props.zigbee2mqttConnected && (
```

Comment:
Please prefer CSS classes (and classnames import).

Reply:
Fixed